# request
---
# response
float32[] joint_angles
float32[] joint_velocities
float32[] joint_torques
float32[] endpoint_geometry
float32[] endpoint_velocity
